apps/judicial-system/web/src/routes/Shared/Cases/PrisonCases.tsx

```diff
@@ -214,7 +215,17 @@ export const PrisonCases: FC = () => {
               cell: (row) => <DefendantInfo defendants={row.defendants} />,
             },
             {
-              cell: (row) => <ColumnCaseType type={row.type} />,
+              cell: (row) => (
+                <CaseTag
```

Comment:
Nit: This is the exact same tag component as above. We could consider extracting it for `indictmentRulingDecision` like we do for `TagAppealState` and `TagCaseState`.